shopify_python/google_styleguide.py

```diff
@@ -251,3 +256,48 @@ def __use_cond_expr(self, node):  # type: (astroid.If) -> None
                 else_body_name = node.orelse[0].targets[0].name
                 if if_body_name == else_body_name:
                     self.add_message('cond-expr', node=node)
+
+    def __lambda_func(self, node):  # type: (astroid.Lambda) -> None
+        """Prefer Operator Function to Lambda Functions"""
+
+        if isinstance(node.body, astroid.UnaryOp):
```

Comment:
You probably want to make sure that the unary operation's operand is just a variable with the same id as the lambda's operand (e.g. you shouldn't recommend replacing `lambda a: not (a + 3)`